tests/tests/features/remote-ssh-portal.yaml

```diff
@@ -0,0 +1,75 @@
+- name: "{{ testname }} - init git, add files, commit, git push"
+  hosts: localhost
+  serial: 1
+  vars:
+    git_files: "node{{ node_version }}/"
+  tasks:
+  - include: ../../tasks/git-init.yaml
+  - include: ../../tasks/git-add-commit-push.yaml
+
+- name: "{{ testname }} - api deployEnvironmentBranch on {{ project }}, which should deploy the first commit"
+  hosts: localhost
+  serial: 1
+  vars:
+    branch: "{{ branch }}"
+    project: "{{ project }}"
```

Comment:
```suggestion
    branch: "{{ branch }}"
    project: "{{ project }}"
    namespace: "{{ project | regex_replace('_', '-') }}-{{ branch | regex_replace('/', '-') }}"
```

Reply:
In https://github.com/uselagoon/lagoon/pull/3039 we need to pass the {{ namespace }} to the deploy tasks, hence this change

Reply:
Thanks, I applied this change